frontend/src/screens/History/History.tsx

```diff
@@ -44,28 +18,28 @@ const History = () => {
         <div className={styles.overviewTiles}>
           <div className={styles.overviewSide}>
             <div className={`${styles.tile} ${styles.overviewLeft}`}>
-              <p className={styles.workoutsNumber}>50</p>
+              <p className={styles.workoutsNumber}>{totalWorkouts}</p>
               <p className={styles.tileHeroText}>Workouts performed so far</p>
             </div>
           </div>
 
           <div className={styles.overviewSide}>
             <div className={styles.overviewUpper}>
               <div className={`${styles.tile} ${styles.timeTile}`}>
-                <p className={styles.timeTileHours}>150</p>
+                <p className={styles.timeTileHours}>{totalTimeWorkingout}</p>
                 <p className={styles.timeTileText}>
                   Total Hours Spent Working Out
                 </p>
               </div>
               <div className={styles.tile}>
                 <p className={styles.tileText}>
-                  {"Bicep Curl"} is your favorite exercise
+                  {favoriteExercise} is your favorite exercise
                 </p>
               </div>
             </div>
             <div className={`${styles.tile} ${styles.overviewBottom}`}>
               <div className={`${styles.streakTile} ${styles.streakLeft}`}>
-                You've logged workouts
+                You've Logged Workouts
```

Comment:
nit: do we want to keep this as sentence case (what you had before)? the other phrases on this page are sentence case, so we should probably stick to one for consistency

Reply:
I honestly wasn't sure what case that should be. I just thought it looked nice this way. I can change it to be more consistent with the res of the page

Reply:
Meta is migrating from capital cases to sentence cases, so I would assume sentence case is preferred from CD's perspectives.

Reply:
Oh, that's really interesting to know. Thanks!